src/components/Breed/ImagesGallery.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import classes from "./ImagesGallery.module.css";
+
+const ImageGallery = (props) => {
+  return (
+    <React.Fragment>
```

Comment:
You can shorten it by using `<> something </>` instead of `<React.Fragment> something </React.Fragment>`

Reply:
This is done